### PR TITLE
New package: arc-themes-20160605

### PR DESCRIPTION
**Pull request by vjon**

### CODE REVIEW

File: srcpkgs/arc-themes/template [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'arc-themes'
+pkgname=arc-themes
+version=20160605
+revision=1
+noarch=yes
+build_style=gnu-configure
+configure_script="./autogen.sh"
+configure_args="--prefix=/usr"
```
Comment (Vaelatern) [inline srcpkgs/arc-themes/template]: not needed

File: srcpkgs/arc-themes/template [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+hostmakedepends="autoconf automake pkg-config"
```
Comment (Vaelatern) [inline srcpkgs/arc-themes/template]: `automake pkg-config`

**Comment by vjon** [comment]
I've fixed the unecessary dependancy, but I'm not sure what you mean by that first comment. Are you saying the entire section (first 8 lines) of the template file can be removed?

File: srcpkgs/arc-themes/template (hunk 1 above)
Comment (vjon) [inline srcpkgs/arc-themes/template]: In regards to the "configure_script" and "configure_args" lines, xbps-src fails during do_configure() when it attempts to run "./configure" in the project directory.

Comment (Vaelatern) [inline srcpkgs/arc-themes/template]: I repeat. Do not set `configure_args` unless you can provide a build log showing it does not work.

Comment (vjon) [inline srcpkgs/arc-themes/template]: Ah, my bad, I was misunderstanding the intention of your original comment. It's fixed now.

**Pull request closed without merging**